docs/dietpi_sbc.md

```diff
@@ -39,7 +39,17 @@ Quick intro:
         <tr>
             <td align="left"><a class="table"><strong>Raspberry Pi - ALL models</strong></a></td>
             <td align="left">
-                <strong>Unique DietPi image available for ALL Raspberry Pi models: Raspberry Pi 4 Model B, Raspberry Pi 3 Model A+ / B / B+, Raspberry Pi 2 Model B, Raspberry Pi 1 Model A+ / B+, Raspberry Pi Zero W</strong>: <a href="https://dietpi.com/downloads/images/DietPi_RPi-ARMv6-Buster.7z">Download image</a></li>
+                <strong>Unique DietPi image available for ALL Raspberry Pi models: </strong><a href="https://dietpi.com/downloads/images/DietPi_RPi-ARMv6-Buster.7z">Download image</a>
+                <strong>
+                <ul>
+                  <li>Raspberry Pi 400</li>
+                  <li>Raspberry Pi 4 Model B</li>
```

Comment:
If we really want to have all in the list:
```
Raspberry Pi Compute Module 4
Raspberry Pi Compute Module 3 / 3+
Raspberry Pi Compute Module 1
```
Or shortened to the SoCs they match to:
```
Raspberry Pi 4 Model B / CM 4
Raspberry Pi 3 Model A+ / B / B+ / CM 3 / CM 3+
Raspberry Pi 1 Model A / A+ / B / B+ / CM 1
```
Raspberry Pi 400 is actually a Raspberry Pi 4 SoC as well, but as it's a new product, it's nice to have it a little outstanding positioned.

Reply:
Ok, added the CMs.